erts/emulator/beam/external.c

```diff
@@ -3775,123 +3705,6 @@ store_in_vec(TTBEncodeContext *ctx,
     }
 }
 
-static byte *
-begin_hopefull_data(TTBEncodeContext *ctx, byte *ep)
```

Comment:
You should also be able to delete the actual transcoding of EXPORT_EXT and BIT_BINARY_EXT in `transcode_dist_obuf()`. I think you can remove the entire
`if (~dflags & hopefull_flags) {
   ...
}
`

Reply:
Done.